api/v1beta1/helmrepository_types.go

```diff
@@ -150,6 +150,7 @@ func (in *HelmRepository) GetInterval() metav1.Duration {
 // +genclient
 // +genclient:Namespaced
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=hr
```

Comment:
This conflicts with the `HelmRelease` from the helm-controller.

Reply:
I see, do you have any recommendation or shall I skip this all together?

Reply:
Bit tricky, as we are running out of two letter acronyms and both omitting it or using another acronym would create a divergence.

What if we just shorten the `repository` part for all? So `gitrepo` and `helmrepo`?

Reply:
Fair enough, that sounds good. Just to confirm, are you suggesting to change `gr` to `gitrepo` or keep both around?

Reply:
Think just `gitrepo` should be OK to prevent having to bring bad news to `hr` suggestions in the future because people think it is "missing".

Reply:
My original intent was to keep it similar to how `ks` is a shortname for `kustomizations`, but will update it as discussed.